core/connection/manager_test.go

```diff
@@ -568,6 +568,10 @@ type mockP2PChannel struct {
 	lock   sync.Mutex
 }
 
+func (m *mockP2PChannel) Fd() int {
```

Comment:
```suggestion
func (m *mockP2PChannel) FD() int {
```

Reply:
I wanted to keep method the same as in go std. `func (f *File) Fd() uintptr`